docs/DeveloperGuide.md

```diff
@@ -6,32 +6,45 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+1. The following sequence diagram shows how the Bye Command works.
+![](ByeCommand.png)
 
 ## Product scope
 ### Target user profile
 
-{Describe the target user profile}
+For airlines counter staff who are fast typist to easily manage flight bookings.
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+Given that the pandemic is ending, more and more people are resuming air travel. <br>
+Therefore, airlines counter staff need a fast an efficient way to manage the many flight bookings.
```

Comment:
This is useful information to the readers. Maybe your team can shift it to the top and add a brief summary, or combine both together to mention what is the purpose behind your application.